src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -559,8 +559,11 @@ export default class DefaultAudioVideoController implements AudioVideoController
         this.logger.info('canceled retry');
       }
     );
-
     if (!willRetry) {
+      errorMessage &&
```

Comment:
How about logging this error message in `handleMeetingSessionStatus` where we log several messages? You can do the following:

```
const willRetry = this.reconnect(status, errorMessage);
if (!willRetry) {
  // Log here
}
return willRetry;
```

Reply:
Yeah, updated, it's more cleaner this way.